src/main/java/List.java

```diff
@@ -0,0 +1,102 @@
+public class List {
+
+    private final int MAX_INSTRUCTION = 100;
+    private Task[] instruction = new Task[MAX_INSTRUCTION];
+    private int index = 0; // Array index
+    private boolean isEmpty = true; // To check if Instruction array is empty
+
+    public void printList() {
+        System.out.println("----------------------------------------");
```

Comment:
Consider making System.out.println("----------------------------------------"); into a method?

Reply:
yeah actually its a good idea! thanks for the suggestion!